R/brm.R

```diff
@@ -486,6 +485,83 @@ brm <- function(formula, data, family = gaussian(), prior = NULL,
   empty <- as_one_logical(empty)
   rename <- as_one_logical(rename)
 
+  # this check is to allow other tests to continue doing what they were expected to do
+  #   when function was called with data parameter missing. We avoid an early
+  #   fail by adding this conversion to d for in our hash function
+  if(missing(data)){
+    d<- NULL
```

Comment:
is this if-statement necessary here? Can we make it more clear? Can we perhaps just default data = NULL? (you would need to check if the errors are still informative)